test/test_diamond.py

```diff
@@ -54,6 +54,19 @@ class Tests(unittest.TestCase):
 VEKKYGLCGVKPSMEVVKDIATEVTLYGLEQYDEYPALLEDHFGGSQRAGVTAAAAGCS
 VAFATGNSNAGINGWYLSQILHKEYHSRLGFYGYDLQDQCGAANSLSIRSDEGLLHECRG'''
 
+    def assertEquivalent(self, list1, list2):
+        pairs = [(item1, item2) for item1, item2 in zip(list1, list2)]
+
+        for res1, res2 in pairs:
+            length = len(res1)
+            self.assertEqual(
+                [res1[i] for i in range(0, length) if not i in [9, 10]],
+                [res2[i] for i in range(0, length) if not i in [9, 10]])
+
+            self.assertAlmostEqual(float(res1[9]), float(res2[9]), 20)
+            self.assertAlmostEqual(float(res1[10]), float(res2[10]), -2)
```

Comment:
Currently, this will fail if the diamond bitscore is <1053. If the rounding is changed to -3, it will fail at <603.